addons/point_of_sale/views/pos_config_view.xml

```diff
@@ -334,7 +334,7 @@
                                     Payment methods available
                                 </div>
                                 <div class="content-group mt16">
-                                    <field name="payment_method_ids" colspan="4" nolabel="1" context="{}" widget="many2many_tags"/>
+                                    <field name="payment_method_ids" colspan="4" nolabel="1" context="{}" widget="many2many_tags" required="1"/>
```

Comment:
as you are modifying this line, could you also remove 'context="{}" ' which is useless